src/dmd/objc.d

```diff
@@ -319,12 +465,55 @@ extern(C++) private final class Supported : Objc
 
     override void setObjc(ClassDeclaration cd)
     {
+        static bool isExtern(Dsymbols* members)
+        {
+            bool result = true;
+
+            members.foreachDsymbol((member) {
+                if (auto attrib = member.isAttribDeclaration)
+                    result = isExtern(attrib.decl);
```

Comment:
This looks like a bug, because it does not stop iteration, and may get set to something different on the next iteration.

Reply:
Hmm, yes.

Reply:
Fixed.